### ISSUE TITLE
Separating v3 tooling list from v2's (cleaning it up)

### ISSUE THREAD

**Issue by akauppi** [coreos/etcd#8007]
The current situation between v2 and v3 in the documentation is confusing. There are separate "libraries, tools and languages" pages, for v3 and v2:

- https://coreos.com/etcd/docs/latest/libraries-and-tools.html
- https://coreos.com/etcd/docs/latest/v2/libraries-and-tools.html

It seems to me that a cleanup of the "v3" page would be in place. It contains entries that seem rather stale, and could be removed. It could clearly indicate which projects aim at supporting the v3 API, since this starts to be a practical use case for coming to the documentation (at least for me). It could show `coreos/jetcd` as more "under construction", since it doesn't provide watch support, test coverage is said to be partial, and (because of these) it's not pushed to Maven Central.

In short, v3 could probably do with a dramatically shorter listing of libraries, tools and languages, than the v2 list.

Ideally, I could reason that the page is simply in the wrong place. Since both v2 and v3 support the v2 API, it should live somewhere outside of the "Communicating with etcd v[23]" roof (if both pages are still maintained).

If v2 page is "frozen", the solution could be to have separate "Libraries, tools and languages (v3 API)", and "... (v2 API)" pages. 

In addition, the "Projects using etcd" seems to be in completely wrong place. It belongs in a git repo README, or something similar? (it's not essential for "communicating with etcd").

One more thing... v2 page sidebar has "langauge bindings" (misspelling).

**Comment by xiang90** [comment]
The etcd repo does not control coreos.com/doc.

Can you move this issue to https://github.com/coreos/docs?

**Comment by akauppi** [comment]
Sure, thank for the pointer @xiang90
